rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -119,6 +119,13 @@ private Config getConfigFromCliArgs() {
             config = entry.getKey().withConfig(config, entry.getValue());
         }
 
+        if (!cliArgs.getParamValueMap().isEmpty()) {
```

Comment:
can you also cover this part with unit tests?

Reply:
Done